### ISSUE TITLE
Jitter when opening the quest «What is width of opening here?»

### ISSUE THREAD

**Issue by deevroman** [streetcomplete/StreetComplete#5813]
**How to Reproduce**
Find the quest "What is width of opening here?" on the map and open it

When zooming in, the animation is ragged. When opening other point quests, the map zooms in smoothly

https://github.com/user-attachments/assets/da266257-c8df-44f5-afe6-6e1e021f37df

https://www.openstreetmap.org/node/3359945255

**Versions affected**
v59.0-alpha1
Android 14 OneUI 6.1 
Samsung A52

p.s. Congratulations on the long-awaited migration to MapLibre!

**Comment by westnordost** [comment]
Yes, this has existed since very long, i.e. since the measure quests were first created. I think the same happens for the opening hours quest, no? It's definitely because of the form, not of MapLibre. I'll have a short look at this...

**Comment by lgommans** [comment]
Interesting, I didn't know there was supposed to be an animation at all. Perhaps I enabled some OS setting to reduce animations since they're usually just artificial delays in workflows as well as battery-eating compared to just rendering the intended view once.

Come to think of it, why does it zoom? I got so used to it that I forgot it bothered me. Can we disable it altogether? If I wanted to have a different zoom level, I'd use this little feature of streetcomplete's called zooming :p

Maybe we could do a quick show of :+1:/:-1: on this comment to indicate whether the zoom also bothers you? So that :+1: is in favor of keeping the user's chosen zoom level. An :eyes: could be used for "I've seen the poll but don't care whether it zooms or not". Of course, anything other than an overwhelming majority would mean it a setting is something to consider; this is just about what you would set that setting to if it existed.

**Comment by mnalis** [comment]
> anything other than an overwhelming majority would mean it a setting is something to consider

@lgommans  Note however that StreetComplete is _very reluctant_ to adding new user preferences. (if there is enough interest to add such option, it might perhaps be an option for [SCEE](https://wiki.openstreetmap.org/wiki/SCEE), though)

- see also: https://github.com/streetcomplete/StreetComplete/issues/1058

**Comment by westnordost** [comment]
It's not really a jitter in the animation but rather the whole app seems to freeze for a short while while the quest form is being initialized. For users that have animation on, this happens while zooming in to the quest.

The zooming is done mostly for two reasons:
1. It is important to show the extents of the element for which this quest is about. Sometimes the view thus has to be zoomed in for that, sometimes it must be zoomed out and sometimes this is not necessary as the zoom is near enough to the zoom necessary to enclose the highlighted element
2. It is important for some quests to show the immediate surrounding, some elements are highlighted. If you are zoomed too far out, you'll not see it

Tldr: i will not remove the zooming
